tests/samplers_tests/tpe_tests/test_truncnorm.py

```diff
@@ -51,9 +51,7 @@ def test_logpdf(a: float, b: float, loc: float, scale: float) -> None:
     [(-np.inf, np.inf), (-10, +10), (-1, +1), (-1e-3, +1e-3), (10, 100), (-100, -10)],
 )
 def test_log_gass_mass(a: float, b: float) -> None:
-    for x in np.concatenate(
-        [np.linspace(0, 1, num=100), np.array([sys.float_info.min, 1 - sys.float_info.epsilon])]
-    ):
-        assert truncnorm_ours._log_gauss_mass(np.array([a]), np.array([b])) == pytest.approx(
-            np.atleast_1d(_log_gauss_mass_scipy(a, b)), nan_ok=True
-        ), f"_log_gauss_mass(x={x}, a={a}, b={b})"
+    a_arr, b_arr = np.array([a]), np.array([b])
+    assert truncnorm_ours._log_gauss_mass(a_arr, b_arr) == pytest.approx(
+        _log_gauss_mass_scipy(a_arr, b_arr), nan_ok=True
+    ), f"_log_gauss_mass(a={a}, b={b})"
```

Comment:
I notices that the `x` was unused, so I removed the loop; hope you don't mind.

Reply:
@jorenham 
Thank you for the catch!

@not522 
Do you think we need to do a follow-up for this?

Reply:
While strictly speaking this change should be in a separate PR, but I think it's acceptable to do it in this PR.